medusa/server/web/home/handler.py

```diff
@@ -1293,8 +1293,8 @@ def check_show_for_language(show_obj, language):
         indexer_api_params = indexerApi(show_obj.indexer).api_params.copy()
         indexer_api_params['language'] = language
         indexer_api_params['episodes'] = False
-        t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
         try:
+            t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
```

Comment:
I'll change the letter. Np.
It's inside the try block. Because the init of tvdb and tmdb do some magic in refreshing the bearer. If an indexer is unavailable, it will throw IndexerUnavailable exception.
I'll try to create one indexer.